### DISCUSSION TITLE
Feature Proposal

### DISCUSSION THREAD

**Discussion by WeidiDeng** [filebrowser/filebrowser#1214] [Ideas]
I think filebrowser needs several features.

1. Quota limit. Limit user storage usage.
2. Favorite and Tag. Easier navigation.
3. Incorporation with aria2 for download and permission to use this feature. Using command to run wget is too limited.

I'll make the pr when I have time.

**Comment by WeidiDeng** [comment]
Also, webdav access method is nice.

**Comment by WinnerWind** [comment]
I don't want to bump this, but these features would be really nice to be incorporated into FB
